docs/tutorials/getting-started-with-flask.rst

```diff
@@ -127,7 +123,7 @@ extension:
     :end-before: [docs:ls-rock-end]
     :dedent: 2
 
-The created rock is about 62MB in size. We will reduce its size later in this
+The created rock is about 65MB in size. We will reduce its size later in this
```

Comment:
This is the amd64 rock size (also checked the reduction size is still roughly 15%)

Reply:
If this image uses ubuntu as the base, that base image changes size constantly, so it might be better to speak in relative terms and mention that it got smaller in the following iteration instead and here mention only that it will